pkg/registries/registries_test.go

```diff
@@ -237,7 +237,7 @@ func TestGenerateConfig(t *testing.T) {
 		{
 			name: "empty mirrors",
 			input: mirrorSet{
-				source:             "registry.access.redhat.com/ubi9/ubi-minimal",
+				source:             "registry.access.redhat.com/ubi9",
```

Comment:
is this change needed?

Reply:
The source should always be a registry, not a reference to a specific image.
The test while functional, was using an impossible value. IMO this change is needed to reduce confusion and set a proper example for how the function should be used.

Reply:
Makes sense.